docs/BLTouch.md

```diff
@@ -1,50 +1,129 @@
 Connecting BL-Touch
 ===================
+
+A **warning** before you start: Avoid touching the BLTouch pin with your bare fingers, since it is
+quite sensitive to finger grease. And if you do touch it, be very gentle, in order to not bend or
+push anything.
+
 Once you've hooked up the BL-Touch servo connector to a `control_pin` according to the BL-Touch
-documentation or of your MCU. Using the original wiring, the yellow wire from the tripple is
+documentation or of your MCU. Using the original wiring, the yellow wire from the triple is
 the `control_pin`, the white wire from the pair is the `sensor_pin`. You need to configure
 these pins according to your wiring with Klipper:
 
+```
+
     [bltouch]
     sensor_pin: P1.24
     #   Pin connected to the BL-Touch sensor pin. This parameter must be
     #   provided.
     control_pin: P1.26
+```
 
-Calibrating the BL-Touch
-========================
+You also most likely need to configure a homing override to first raise the z-axis, then home the x-
+and y-axis, and finally move to the center and home the z-axis. Like this:
 
-### Z Offset
-In order to make klipper work properly you need to tell it in which relation to the nozzle
-the probe is exactly located. Lets start with the `z_offset` - the height of the probe when
-its expanded:
+```
+    [homing_override]
+    gcode:
+        G1 Z10 ; Move up 10mm
+        G28 X Y
+        G1 X166 Y120 F6000 ; Change the X and Y coordinates to the center of your print bed
+        G28 Z
+    set_position_z: 0.0
+```
+
+It's important that the upwards movement is high enough, so that the probe or the nozzle don't hit
+any binder clips, even if the probe pin happens to be in it's lowest state.
 
-Make sure the probe is above the bed by moving the head into an appropriate x/y position:
+Initial tests
+=============
 
-    g0 x0 y0
+Before moving on, verify that the BLTouch is mounted at the correct height, the pin should be
+roughly 2 mm above the nozzle when retracted
 
-Now make klipper aproach the bed until the probe triggers using:
+When you turn on the printer, the BLTouch probe should perform a self-test and move the pin up and
+down a couple of times. Once the self-test is completed, the pin should be retracted and the red LED
+on the probe should be lit. If there are any errors, for example the probe is flashing red or the
```

Comment:
This may also happen if the magnet in the probe slips, see the last section. Please reference this too.

Reply:
Yes, I will